### DISCUSSION TITLE
CR Touch measures 3x per point

### DISCUSSION THREAD

**Discussion by bronsn** [mriscoc/Ender3V2S1#781] [Q&A]
Hey all, do anyone has any idea why my CR Touch (Printer Ender 3V2) is measuring 3 times at any point at auto mesh ?
In the settings i adjust 2 times..
if i change the settings to measure 1 time each point he does, but after i switch back at 2 times each point, he does the same again and measure 3x each point ...
it confused me a bit xD

**Comment by ghost** [comment]
because you have set (by default) Advanced >>> Probe Settings >>> Multiple Probing == 2

Just make it 0 it would be once. But I think this is bad. The CR Touch (sorry, I am newbie still) have some problems (sometimes) and better to use at least 3 probes.

This kind of "voting" technique is similar to used in drive-by-wire in Infinity cars and areoplanes (3 actuators do decisions, 2v1 or 3v0 is decision - unlikely 2 are faluty/decide wrong)

I think this could be closed out.

Please tell me if I helped you.
